install_protoc_from_source.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+
+PROTOC_VERSION="2.6.1"
+
+if [[ ! -d "$HOME/protobuf/lib" ||  "$(protoc --version)" != *${PROTOC_VERSION}* ]]; then
```

Comment:
Likely edge case but what if user needs another version of protobuf installed for another application?